processing.go

```diff
@@ -91,6 +91,10 @@ func Process(mon *Monitor) {
 	//Start all the workers
 	for i := 0; i < workers; i++ {
 		go func() {
+			for _, scannerName := range orderedScanners {
+				scanner := *scanners[scannerName]
```

Comment:
I don't think you want to dereference here. What's the type of `orderedScanners`?

Reply:
`orderedScanners` is just strings corresponding to scanner name. `scanners` is of type `map[string]*Scanner` where Scanner is an interface, and we can't operate on a pointer to an interface.

Reply:
For my own education, I tried this without dereferencing it, and it fails to compile with "scanner.InitPerSender undefined (type *Scanner is pointer to interface, not interface)", which seems to make sense in this context (if it was a Scanner.(*MySQLScanner) then maybe not...